### PR TITLE
Issue with displaying Errors messages

### PR DESCRIPTION
**Pull request by scures**
Fixes: https://github.com/rancher/dashboard/issues/5213

This PR will fixes it by looking if the `label` objects includes a `message` propriety, if so it will stringify and display the Error message.

image

## How to reproduce:

Edit any Pod configuration vía YAML and change any boolean value to something invalid like falsea , this will return a 400 Error and you'll be able to preview it.

### CODE REVIEW

**Review by cnotv — CHANGES_REQUESTED** [review]
Could you define a case to reproduce an error containing a message?

**Comment by scures** [comment]
> Could you define a case to reproduce an error containing a message and reference the issue if exists?

I noticed that any response will an error (validation, network status etc..) will be affected. One way to reproduce would be: 
- Edit any Pod configuration vía YAML and change any boolean value to something invalid like `falsea` , this will return a 400 Error and you'll be able to preview it.

File: components/Banner.vue [hunk 1]
```diff
@@ -29,6 +29,7 @@ export default {
   <div class="banner" :class="{[color]: true, closable}">
     <slot>
       <t v-if="labelKey" :k="labelKey" :raw="true" />
+      <span v-else-if="label.message" v-html="JSON.stringify(label.message)" />
```
[the remark this answers is not in the document]
Comment (cnotv) [reply components/Banner.vue:32]: Correcting my suggestion after discussing and realizing it's not possible to use the optional operator within the template:

In the template
```html
  <span v-else v-html="parsedLabel" />
  ```

As computed variable
```ts
parsedLabel(): string {
  return this.label?.message 
    ? JSON.stringify(this.label.message)
    :  nlToBr(this.label))
}
```

[the remark this answers is not in the document]
Comment (scures) [reply components/Banner.vue]: @cnotv I pushed the changes with the  ` return !(typeof this.label === 'string') ? stringify(this.label) : undefined;` seems the more reasonable solution at this point.

What do you think?

[the remark this answers is not in the document]
Comment (cnotv) [reply components/Banner.vue]: For me it's ok, as I would have to investigate way too much to make it work with the type and we don't want to get stuck now with this.

**Review by cnotv — APPROVED** [review]

**Comment by nwmac** [comment]
@richard-cox Could you give this the once-over - is this something we want to get in for 2.6.4?

**Review by richard-cox — APPROVED** [review]

**Pull request merged**